tex/polyglossia.lua

```diff
@@ -93,6 +92,9 @@ local function load_tibt_eol()
     require('polyglossia-tibt')
 end
 
+-- LaTeX's language register is \count19
+local lang_register = 19
+
```

Comment:
This one is not guaranted. Could we get the latex register interger from latex code add pass it, to this function ?

Reply:
As you can see in `latex.ltx`, `19` is hardcoded in LaTeX format. 
```tex
\count19=0 % allocates \language codes 1, 2, ...
```
I don't think that we can get it from some latex code or lua function.

Reply:
Could we check therefore that \def\newlanguage{\alloc@9\language\chardef\@cclvi} is the same than latex definition ?

Using expl3

Reply:
Parsing `\newlanguage` macro? Hm... that might be possible. 
But I do not think that it will give us cost-benefit efficiency, as the language register has always been `19` since Knuth's plain TeX and we can safely expect it will remain as it is in the future as well.